_sass/main.scss

```diff
@@ -131,6 +131,22 @@ p {
   color: $navy;
 }
 
```

Comment:
Okay to add these styles to the global scope?

Here is how the `a` renders:

![image](https://user-images.githubusercontent.com/4118615/88123966-3b0eb880-cb89-11ea-8dd5-b368afb14339.png)

and here is how `.button-link` renders:

![image](https://user-images.githubusercontent.com/4118615/88124000-4a8e0180-cb89-11ea-92a4-01e7d00611a2.png)

Reply:
Yes, adding that style to the global scope is perfect.